### ISSUE TITLE
Disallow project owners to edit the name after the project was created

### ISSUE THREAD

**Issue by kimorq** [graphprotocol/mission-control-curator#243]
I have seen a few projects that have been challenged for being a duplicate, and as a result the owner has edited them, completely changed all the fields, thus transforming it into a new project.
For instance if I create the Dogecoin project, which already exists, and then this gets challenged, I can then just change all fields of the challenged project (name, logo, cover, description, links) and transform it into another project that is not a duplicate.
With this I get away with creating spam on the network by just paying an edit fee for rebranding my project, whilst honest curators will pay transaction fees for the votes because they will think I am a legit project and they will want to support me.

I think we could avoid this by disallowing the project's name to be changed once created/challenged.

**Comment by yanivtal** [comment]
Yes this is a good one to think about. I'm not 100% sold this is the right solution but something like it might be.
